src/Device/deviceSetup.ts

```diff
@@ -83,6 +83,10 @@ export const prepareDevice =
     ): AppThunk<RootState, Promise<void>> =>
     async dispatch => {
         const onSuccessWrapper = (d: Device) => {
+            logger.info(
+                `Connected to device with the serial number ${device.serialNumber} ` +
```

Comment:
`the device`, but oh well, I was not invited to review :P

Reply:
I'll add it in a separate PR no worries